dist/main.js

```diff
@@ -0,0 +1,44 @@
+const tempe = new TempManager
+const renderer = new Renderer
+
+const loadPage = async function () {
+    let currentTime = new Date(Date.now())
+    await tempe.getDataFromDB()
+    if (tempe.cityData.length > 0) {
+        if ((((currentTime - tempe.cityData[0].updatedAt) / 3600000) % 24) > 3) {
+            await tempe.cityData.forEach(c => tempe.updateCity(c.name))
```

Comment:
if you using loop for an async function, the callback function needs to be async, not the outer:
`tempe.cityData.forEach(asnyc c => await tempe.updateCity(c.name))`